apps/site/assets/css/_location-card.scss

```diff
@@ -99,3 +85,73 @@
   float: left;
   width: 83%;
 }
+
+#get-directions-dropdown {
+  display: flex;
+  text-decoration: none;
+  &:hover {
+    color: $brand-primary;
+    text-decoration: none;
+
+  }
+
+  // do not show Bootstrap's default caret:
+  &::after {
+    display: none;
+  }
+}
+
+.dropdown-item:hover {
+  background-color: $brand-primary-lightest-contrast;
+}
+
+.c-caret {
+  @include caret($color: $brand-primary);
+  margin-left: .3rem;
+
+  &::after {
+    content: $fa-var-angle-down;
+  }
+}
+
+.c-location-card--rsl {
+  @include media-breakpoint-up(sm) {
+    flex: 1 1 30%;
+  }
+}
+.c-location-card--psl {
+  @include media-breakpoint-up(sm) {
+    flex: 0 0 30%;
+  }
+}
+
+.c-finder-results {
+  margin-top: $space-16;
+  padding-top: $space-16;
+}
+
+.c-flex-space-between {
+  display: flex;
+  justify-content: space-between;
+}
+
+.c-finder-section {
+  @include media-breakpoint-down(sm) {
+    display: flex;
+    flex-direction: column;
+  }
+  @include media-breakpoint-down(sm) {
+    .c-suggest-retailer {
+      order: -1;
+    }
+  }
+}
+
+.m-error-message {
+  color: $error-text;
+  padding: 7px;
+}
+
+.m-top-half {
+  max-width: 600px;
+}
```

Comment:
These seem only relevant to this page.. might it be useful to start a `.m-sales-locations` class on the overall container (kind of like how we have `.m-schedule-line` covering most of the line diagram page) and scoping these under that? In any case, we use the "m" to indicate a module (which generally covers a larger area and will have lots of bits in it) and I'm not sure that makes sense for these particular bits of CSS.

Reply:
Ahhh excellent pointer.  For styles that exclusively belong to location cards, but are used in different parts of the app (for example, location cards appear on the stop pages in the "Fares" section") would that be a "c-" or "m-" style?

Reply:
"c-" (for the location card "component"!)

Reply:
Thanks!  I tried to clean up the style names throughout this file, mind taking another close look to see if I took the right approach?